### ISSUE TITLE
Detailed training settings with DANet

### ISSUE THREAD

**Issue by PkuRainBow** [JACKYLUO1991/FaceParsing#15]
Really good code base!

According to the results in Table-1, we find the performance of DANet is also relatively high. So we have some concern on it detailed training settings.

- Do you apply only the Cross-Entropy loss or the combination of (bootstrapped_cross_entropy2d version) Cross-Entropy loss and Dice loss?

- Do you apply the ResNet-101 for both DANet and CE2P? However, your best results are achieved based on ResNet-34, which is very interesting.

- How about their best results on the validation set? According to the results in Table-4, is your best results on validation set 73%? Besides, is it based on single-scale testing or multiple-scale testing + flip testing?

Thanks for your kind help!

**Comment by JACKYLUO1991** [comment]
@PkuRainBow  Sorry to reply late, I am finishing my graduation thesis. 
1. We use a composite loss function, which includes the supervision of the boundary, please check the corresponding paper for details;
2. Yes；
3. We did not verify the indicators of other methods on the validation set. In addition, we did use multiple-scale testing + flip testing.

**Comment by PkuRainBow** [comment]
@JACKYLUO1991 Thanks for your reply. Besides, I am also wondering whether you have applied the Sync-BN in all of your experiments?

**Comment by JACKYLUO1991** [comment]
@PkuRainBow In fact, it does not. Sync-BN is suitable for multi-cards. Our laboratory only configures a single card, so no Sync-BN is used.  In addition, it seems that the official code of CE2P uses Sync-BN.

**Comment by PkuRainBow** [comment]
@JACKYLUO1991 Really interesting! I guess DANet requires at least 4x16G memory. So how do you train the DANet with only a single card?

**Comment by JACKYLUO1991** [comment]
@PkuRainBow Reduce the batch-size appropriately, I remember it seems to be 2 or 4.
